### DISCUSSION TITLE
Two keystroke modal

### DISCUSSION THREAD

**Discussion by jgarte** [meow-edit/meow#449] [Q&A] [answered]
Is it possible to do something like `gd` in normal mode in order to execute `xref-find-definitions`?

That is, two keystrokes: `g` followed by `d`.

I realize I can bind it to a single keystroke like `t` but I just wanted to ask about how I can go about implementing this using meow APIs.

**Comment by DogLooksGood** [comment, marked as answer]
```
(meow-define-keys 'normal
  '("z a" . beginning-of-buffer)
  '("z b" . end-of-buffer))
```
Do it this way. It's equivalent to
```
(define-key meow-normal-mode (kbd "z a") 'beginning-of-buffer)
(define-key meow-normal-mode (kbd "z b") 'end-of-buffer)
```

The keymaps used in meow are just plain keymaps. The helper functions are not really necessary, pick the way you like.

**Reply by jgarte** [reply]
Awesome! Thanks, I'll do that.
